src/overlays/actors/ovl_En_Ik/z_en_ik.c

```diff
@@ -13,44 +13,76 @@
 
 typedef void (*EnIkDrawFunc)(struct EnIk*, PlayState*);
 
+#define ARMOR_BROKEN (1 << 0)
+
+typedef enum {
+    /* 0x00 */ IK_ACTION_INIT,
+    /* 0x01 */ IK_ACTION_1,
+    /* 0x02 */ IK_ACTION_2,
+    /* 0x03 */ IK_ACTION_3,
+    /* 0x04 */ IK_ACTION_4,
+    /* 0x05 */ IK_ACTION_5
+} EnIkAction;
+
+typedef enum {
+    /* 0x00 */ IK_DRAW_NOTHING,
+    /* 0x01 */ IK_DRAW_1,
+    /* 0x02 */ IK_DRAW_2
+} EnIkDrawMode;
+
+typedef enum {
+    /* 0 */ IK_TYPE_NABOORU,
+    /* 1 */ IK_TYPE_SITTING,
```

Comment:
```suggestion
    /* 1 */ IK_TYPE_SILVER,
```
Does this make sense for OoT? It would match what you have in MM.